### ISSUE TITLE
Potential interest in helping out on this

### ISSUE THREAD

**Issue by rob-p** [dcjones/PoleeClassifier.jl#1]
Hi again @dcjones!

> Lmk if this sounds cool and you want to colab, especially if you have some nice data.

The idea behind this repo so far sounds super-interesting to me.  I'm not sure I can say I (or the other ppl in my group who are likely interested) have a nice idea; but we do have some potentially interesting ideas in this space.  I think we'd be excited to potentially collaborate on this.  I also wanted to ping @hiraksarkar — since I'm like 99% sure he'd be interested as well ;P.

**Comment by dcjones** [comment]
I would love to talk about this. Just to sketch out what I'm playing around with: Suppose we have some training data consisting of labels (e.g. cell type) `y`, and reads `r` and we want to train a neural network `f` with weights `W` to predict the labels from reads.

We could estimate the unobserved expression `x`, say with the posterior mean `E[x]`, then train the neural network
```
   argmin_{W} loss(y, f(E[x]; W))
```

That's pretty standard, but my thought is that with approximate likelihood we could move that expectation outward and instead do
```
   argmin_{W} E[loss(y, f(x; W))]
```

where expectations are over `x ~ p(x|r)`. Since minimizing expected loss is the more theoretically correct thing to do, this may have some have some benefit.

I'm playing around with classifying gtex brain tissues, and had some mildly promising results, but its probably not an ideal benchmark as its relatively deeply sequenced and most the tissues are pretty dissimilar. Whether it works better than point estimates also seems a somewhat contingent on the architecture of the neural network, which I'm still trying to wrap my head around.

**Comment by blahah** [comment]
Stumbling over this thread suggesting that you two might collab gives me hope.
